tests/e2e/specs/modules/analytics/setup-no-account-no-tag.test.js

```diff
@@ -92,9 +92,10 @@ describe( 'setting up the Analytics module with no existing account and no exist
 
 		await Promise.all( [
 			expect( page ).toClick( 'button', { text: /configure analytics/i } ),
-			page.waitForSelector( '.googlesitekit-publisher-win__title' ),
+			page.waitForNavigation(),
 		] );
 
+		await page.waitForSelector( '.googlesitekit-publisher-win__title' );
```

Comment:
@aaemnnosttv Can you clarify why you changed this (I'm not that familiar)? This may actually be related to one of the test failures in #981.